homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -244,7 +244,7 @@ async def async_volume_up(self) -> None:
         """Send volume up command."""
         volume = await self.fs_device.get_volume()
         volume = int(volume or 0) + 1
-        await self.fs_device.set_volume(min(volume, self._max_volume))
+        await self.fs_device.set_volume(min(volume, self._max_volume or 1))
```

Comment:
Used `1` as default if `self._max_volume` instead set since that's the min value for `volume` up anyway.